core/runtime/src/main/java/io/quarkus/runtime/util/StringUtil.java

```diff
@@ -13,6 +13,20 @@ public final class StringUtil {
     private StringUtil() {
     }
 
+    public static String changePrefix(String name, String oldPrefix, String newPrefix) {
+        if (!name.startsWith(oldPrefix)) {
+            return name;
+        }
+        // don't use here String::replaceFirst because it uses regex
```

Comment:
Ah f**k. I thought `replaceFirst` was like `replace` and not using regexp :/

Reply:
Yeah, I was surprised as well! Said that, you know that this change I'm pushing is a drop in the ocean which value is mostly on benchs as far as I can tell :/ 
And likely, the renaming will goes away in few releases too